templates/data/spark-on-kubernetes.html

```diff
@@ -0,0 +1,321 @@
+            <hr class="p-rule" />
+            <p>
+              <a class="p-button--positive" aria-controls="data-spark-modal" href="/contact-us">Request a demo</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <hr class="p-rule" />
+    <div class="col-6 col-medium-3">
+      <div class="p-block">
+        <h2>Learn more</h2>
+      </div>
+    </div>
+    <div class="col-6 col-medium-3 col-start-medium-4">
+      <div class="p-block">
+        <div class="row">
+          <div class="col-2 col-medium-3">
+            <h3 class="p-heading--5">Watch</h3>
+          </div>
+          <div class="col-4 col-medium-3">
+            <p>
+              <a href="https://ubuntu.com/engage/spark-security-webinar">Webinar on Spark security for big data operations</a>
+            </p>
+            <p>Uncover best practices and practical steps to Spark security.</p>
+          </div>
+        </div>
+      </div>
+      <div class="p-block">
+        <hr class="p-rule" />
+        <div class="row">
+          <div class="col-2 col-medium-3">
+            <h3 class="p-heading--5">Download</h3>
+          </div>
+          <div class="col-4 col-medium-3">
+            <p>
+              <a href="https://ubuntu.com/engage/enterprise-data-management-big-data">Guide to implementing big data solutions with open source</a>
+            </p>
+            <p>A security-maintained and fully supported solution for Apache Spark on Kubernetes.</p>
+          </div>
+        </div>
+      </div>
+      <div class="p-block">
+        <hr class="p-rule" />
+        <div class="row">
+          <div class="col-2 col-medium-3">
+            <h3 class="p-heading--5">Discover</h3>
+          </div>
+          <div class="col-4 col-medium-3">
+            <p>
+              <a href="https://ubuntu.com/kubernetes">Canonical Kubernetes</a>
+            </p>
+            <p>Learn how our our Kubernetes solutions can accelerate data lake deployments in hybrid cloud environments.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<hr class="p-rule u-no-margin--bottom is-fixed-width" />
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-12">
+      <div class="p-block">
+        <p class="p-heading--2 u-no-margin--bottom"><a href="https://ubuntu.com/engage/spark-reference-architecture-for-big-data">Download the reference architecture guide&nbsp;&rsaquo;</a></h2>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row">
+    <hr class="p-rule">
+    <div class="col-6">
+      <p>Apache&reg;, Apache Spark, Spark&reg;, and the Spark logo are either registered trademarks or trademarks of the Apache Software Foundation in the United States and/or other countries.</p>
+    </div>
+  </div>
+</section>
+
```

Comment:
My bad, just realized why the modal isn't working 😅 
```suggestion
<script defer src="{{ versioned_static('js/modals.js') }}"></script>

{% with %}
  {% set returnURL = "https://canonical.com/data/spark#success" %}
  {% include "data/_form-data-spark.html" %}
{% endwith %}
```